app/src/main/java/com/vinh/dictionary_1/utis/Constant.java

```diff
@@ -9,4 +9,7 @@
    public static final String REGEX_WORD_PHONETIC = "(\\/[^></�]+\\/)|(\\[[^></�]+\\])";
    public static final String DB_NAME_ENGLISH_VIETNAMESE = "ev.db";
    public static final String DB_NAME_VIETNAMESE_ENGLISH = "ve.db";
+   
+   public static final int DRAWER_HOME_OPEN_CONTENT_DESC_RES = 1;
```

Comment:
ERROR: 'member def modifier' have incorrect indentation level 3, expected level should be 4.